README.md

```diff
@@ -22,11 +24,11 @@ repositories {
 }
 
 dependencies {
-	compile ('com.github.uphold:uphold-sdk-android:0.0.2@aar') {
+	compile ('com.github.uphold:uphold-sdk-android:0.0.3@aar') {
 	    transitive = true
 	}
 	// Change to:
```

Comment:
BTW, we could simplify this by adding the comment to the `compile` above and have the comment a little more descriptive.